### ISSUE TITLE
[a11y] Cannot save form since name is not detected

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1427]
I cannot save a locality, even one that already exists, since the name is not being detected. 

**To Reproduce**
1. Go to unitec, /specify/view/locality/139216/`
2. Try to save the Locality
3. See error

**Screenshots**
![msedge_zR50OHyB2E](https://user-images.githubusercontent.com/37256050/167212327-d1bea64d-9031-4b7f-b011-9c03ebcbea04.png)

Windows 10, Edge
